src/manage_sql.c

```diff
@@ -15490,37 +15490,41 @@ update_nvti_cache ()
   nvti_cache = nvtis_new ();
 
   init_iterator (&nvts,
-                 "SELECT oid, name, family, cvss_base, tag, solution, solution_type FROM nvts;");
+                 "SELECT nvts.oid, nvts.name, nvts.family, nvts.cvss_base,"
+                 "       nvts.tag, nvts.solution, nvts.solution_type,"
+                 /*      7 */
```

Comment:
I think this comment should be removed or replaced with one that briefly explains how the iterator combines the normal NVT info with the refs and why.

Reply:
Good point, done in https://github.com/greenbone/gvmd/pull/826/commits/2e484026827f76d3e4954780a3cae4d2fce6c8aa.